Makefile

```diff
@@ -53,12 +53,13 @@ lint:
 
 .PHONY: build
 build: deps
-	$(GO) install -x -ldflags "$(GOBUILD_LDFLAGS)" $(ALL_PACKAGES)
+	GOOS=linux $(GO) build -o gfmrun-linux-amd64-$(VERSION_VALUE)  -x -ldflags "$(GOBUILD_LDFLAGS)" ./cmd/gfmrun/main.go
+	GOOS=darwin $(GO) build -o gfmrun-darwin-amd64-$(VERSION_VALUE) -x -ldflags "$(GOBUILD_LDFLAGS)" ./cmd/gfmrun/main.go
+	GOOS=windows $(GO) build -o gfmrun-windows-amd64-$(VERSION_VALUE).exe  -x -ldflags "$(GOBUILD_LDFLAGS)" ./cmd/gfmrun/main.go
```

Comment:
Also, the version is implied in the release tag, do we really want to duplicate it in the filenames?

Reply:
I think this would save the user from confusion if they had multiple versions of the binary on their system.